graphql-java-support/src/test/resources/com/apollographql/federation/graphqljava/schemas/directives-output.graphql

```diff
@@ -0,0 +1,32 @@
+schema {
+  query: Query
+}
+
+"Directs the executor to include this field or fragment only when the `if` argument is true"
+directive @include(
+    "Included when true."
+    if: Boolean!
+  ) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
+
+"Directs the executor to skip this field or fragment when the `if`'argument is true."
+directive @skip(
+    "Skipped when true."
+    if: Boolean!
+  ) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
+
+"Marks the field or enum value as deprecated"
+directive @deprecated(
+    "The reason for the deprecation"
+    reason: String = "No longer supported"
+  ) on FIELD_DEFINITION | ENUM_VALUE
```

Comment:
Not quite sure why these started showing up in the SDL - https://github.com/graphql-java/graphql-java/pull/1723 maybe?

Reply:
I believe it's from graphql-java/graphql-java#1609 , specifically the removal of
```
        // we don't print the standard directives that always ship with graphql-java
        List<String> standardDirectives = Arrays.asList(
                "skip",
                "include",
                "defer",
                "deprecated");
```